internal/viewmodel/message.go

```diff
@@ -1,21 +1,52 @@
 package viewmodel
 
+import (
+	"backend/internal/model"
+)
+
 type BaseMessageView struct {
-	TelegramMessageId int64     `json:"telegram_message_id"`
+	TelegramMessageID int64     `json:"telegramMessageId"`
 	By                string    `json:"by"`
-	MessageBody       string    `json:"message_body"`
+	MessageBody       string    `json:"messageBody"`
 	Timestamp         string    `json:"timestamp"`
-	HotelStaffId      *uint     `json:"hotel_staff_id,omitempty"`
-	HotelStaff        *UserView `json:"hotel_staff,omitempty"`
-	RequestQueryId    uint      `json:"request_query_id,omitempty"`
+	HotelStaffID      *uint     `json:"hotelStaffId,omitempty"`
+	HotelStaff        *UserView `json:"hotelStaff,omitempty"`
+	RequestQueryID    uint      `json:"requestQueryId,omitempty"`
```

Comment:
Up for discussion - format of json:

https://www.quora.com/Should-I-use-camelCase-or-underscore-for-JSON

Reply:
@seelengxd @chloeelim, any preferences?

Reply:
https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase

i dont have particularly strong opinion on this but given that both go and js use camelCase and based on that link, I think camelCase may be nicer 🤔